dretesteth.sh

```diff
@@ -112,6 +112,9 @@ if [ "$SCRIPT_NAME" = "dtf.sh" ]; then
             if [ "$var" = "nimbus" ]; then
                 binpath="/bin/evm_nimbus"
             fi
+            if [ "$var" = "nethermind" ]; then
+                binpath="/nethermind/tools/Evm/Evm/bin/Debug/net8.0/Evm"
```

Comment:
to save space I copy built binaries into bin folder inside docker and remove all src and build files that are no longer required for it to run. making docker image smaller.